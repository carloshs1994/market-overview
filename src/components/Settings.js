import React from 'react';

const Settings = () => (
  <main>
    <h1>Settings Under construction...</h1>
  </main>
);

export default Settings;
