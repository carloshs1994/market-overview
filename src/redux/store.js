import { createStore, combineReducers, applyMiddleware } from 'redux';
import logger from 'redux-logger';
import thunk from 'redux-thunk';
import homeReducer from './home/home';
import detailsReducer from './details/details';

const reducers = combineReducers({
  homeReducer,
  detailsReducer,
});

const store = createStore(
  reducers,
  applyMiddleware(logger, thunk),
);

export default store;
