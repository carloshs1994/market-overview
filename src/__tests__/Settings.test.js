import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import Settings from '../components/Settings';
import store from '../redux/store';

test('matches voice search component snapshot', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <Settings />
    </Provider>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
