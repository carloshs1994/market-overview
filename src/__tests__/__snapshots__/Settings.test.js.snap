// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches voice search component snapshot 1`] = `
<main>
  <h1>
    Settings Under construction...
  </h1>
</main>
`;
